sdk/core-rest/core-client/src/sendRequest.ts

```diff
@@ -0,0 +1,77 @@
+import {
+  createHttpHeaders,
+  createPipelineRequest,
+  HttpMethods,
+  Pipeline,
+  RawHttpHeaders,
+} from "@azure/core-rest-pipeline";
+import { getCachedDefaultHttpsClient } from "./clientHelpers";
+import { RequestParameters } from "./pathClientTypes";
+import { HttpResponse } from "./common";
+
+/**
+ * Helper function to send request used by the client
+ * @param method method to use to send the request
+ * @param url url to send the request to
+ * @param pipeline pipeline with the policies to run when sending the request
+ * @param options request options
+ * @returns returns and HttpResponse
+ */
+export async function sendRequest(
+  method: HttpMethods,
+  url: string,
+  pipeline: Pipeline,
+  options: RequestParameters = {}
+): Promise<HttpResponse> {
+  const httpClient = getCachedDefaultHttpsClient();
+
+  const headers = createHttpHeaders({
+    accept: "application/json",
```

Comment:
should not that be configurable? or is this the only thing supported in this version?

Reply:
That's a good point, maybe we can take it as part of the options